index.js

```diff
@@ -55,7 +55,8 @@ module.exports = {
     'no-undef': 2,
     'no-underscore-dangle': 0,
     'no-unused-expressions': 0,
-    'no-unused-vars': 0,
+    // Disallow declaration of variables that are not used in the code.
+    'no-unused-vars': [2, { vars: 'local', args: 'after-used' }],
```

Comment:
According to the docs, `after-used` is the default setting, so I don't think we need to set it again.